tidb-global-sort.md

```diff
@@ -0,0 +1,87 @@
+---
+title: TiDB Global Sort
+summary: Learn the use cases, limitations, usage, and implementation principles of the TiDB Global Sort.
+---
+
+<!-- markdownlint-disable MD029 -->
+<!-- markdownlint-disable MD046 -->
+
+# TiDB Global Sort
+
+> **Warning:**
+>
+> This feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> Currently, this feature is only applicable to TiDB Dedicated clusters. You cannot use it on TiDB Serverless clusters.
+
+</CustomContent>
+
+## Overview
+
+The TiDB Global Sort feature enhances the stability and efficiency of data import and DDL (Data Definition Language) operations. It serves as a general operator in the [distributed execution framework](/tidb-distributed-execution-framework.md), providing a global sort service on cloud.
+
+The Global Sort feature can be easily extended to support multiple shared storage interfaces such as S3 and POSIX, enabling seamless integration with different storage systems. This flexibility enables efficient and adaptable data sorting for various use cases.
```

Comment:
```suggestion
The Global Sort feature currently only supports using Amazon S3 as cloud storage. In future releases, it will be extended to support multiple shared storage interfaces, such as OSIX, enabling seamless integration with different storage systems. This flexibility enables efficient and adaptable data sorting for various use cases.
```